ERCS/erc-725.md

```diff
@@ -13,15 +13,15 @@ requires: 165, 173
 
 ## Abstract
 
-The following describes two standards that allow for a generic data storage in a smart contract and a generic execution through a smart contract. These can be used separately or in conjunction and can serve as building blocks for smart contract accounts, upgradable metadata, and other means.
+The following describes two standards that allow for generic data storage in a smart contract and a generic execution through a smart contract. These can be used separately or in conjunction and can serve as building blocks for smart contract accounts, upgradable metadata, and other means.
 
 ## Motivation
 
 The initial motivation came out of the need to create a smart contract account system that's flexible enough to be viable long-term but also defined enough to be standardized. They are a generic set of two standardized building blocks to be used in all forms of smart contracts.
 
-This standard consists of two sub-standards, a generic data key/value store (`ERC725Y`) and a generic execute function (`ERC725X`). Both of these in combination allow for a very flexible and long-lasting account system. The account version of `ERC725` is standardized under `LSP0-ERC725Account`.
+This standard consists of two sub-standards, a generic data key/value store (Y) and a generic execute function (X). Both of these in combination allow for a very flexible and long-lasting account system.
```

Comment:
Your original names (`ERC725Y` and `ERC725X`) are better than just `Y` and `X`. Instead of calling them sub-standards, just call them interfaces. It's pretty normal to have a few different interfaces in the same ERC. To satisfy the linter, you just need to use backticks (\`).